src/traces/parcoords/plot.js

```diff
@@ -0,0 +1,57 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var parcoords = require('./parcoords');
+var Plotly = require('../../plotly');
+var Lib = require('../../lib');
+
+module.exports = function plot(gd, cdparcoords) {
+
+    var fullLayout = gd._fullLayout;
+    var root = fullLayout._glcontainer.node();
+    var data = cdparcoords.map(function(d) {return d[0];});
+
+    var filterChangedCallback = function(value) {
+
+        var restyleStyle = ['deepProp', 'object', 'replot', 'custom'][3];
```

Comment:
Yes I still need to remove most of these (leftover from comparing event generation options).